### PR TITLE
Documentation for Darknet model.

### PR DESCRIPTION
**Pull request by kartikdutt18**
Hey everyone, This PR aims to complete the readme and add documentation for Darknet models which will be added in #20.

### CODE REVIEW

**Comment by kartikdutt18** [comment]
I'm working on completing the wiki page [here](https://github.com/kartikdutt18/models/wiki/Models-Tutorial).

File: README.md [hunk 1]
```diff
@@ -193,49 +192,17 @@ Augmentation augmentation({"horizontal-flip", "resize : (64, 64)"}, 0.2);
...
-
-_(This section also needs some cleanup once we know what we're keeping and what
-we're not keeping.)_
+### 6. Supported Models
 
 Currently model-zoo project has the following models implemented:
+|  **Model** | **Usage** | **Available Weights** | **Paper** |
+| --- | --- | --- | --- |
+|  Darknet&nbsp;19 | DarkNet<NegativeLogLikelihood<>, HeInitialization, 19>&nbsp;darknet19({imageDepth, imageWidth, imageHeight}, numClasses)| Cifar-10 |[YOLO9000](https://pjreddie.com/media/files/papers/YOLO9000.pdf)|
+|  Darknet&nbsp;53 | DarkNet<NegativeLogLikelihood<>, HeInitialization, 53>&nbsp;darknet19({imageDepth, imageWidth, imageHeight}, numClasses)| Cifar-10 |[YOLOv3](https://pjreddie.com/media/files/papers/YOLOv3.pdf)|
```
Comment (KimSangYeon-DGU) [inline README.md:201]:
```suggestion
|  Darknet&nbsp;19 | DarkNet<NegativeLogLikelihood<>, HeInitialization, 19>&nbsp;darknet19({imageDepth, imageWidth, imageHeight}, numClasses)| CIFAR-10 |[YOLO9000](https://pjreddie.com/media/files/papers/YOLO9000.pdf)|
|  Darknet&nbsp;53 | DarkNet<NegativeLogLikelihood<>, HeInitialization, 53>&nbsp;darknet19({imageDepth, imageWidth, imageHeight}, numClasses)| CIFAR-10 |[YOLOv3](https://pjreddie.com/media/files/papers/YOLOv3.pdf)|
```

Comment (kartikdutt18) [reply README.md:201]: Ahh, makes sense. Thanks a lot.

**Comment by saksham189** [comment]
I think we should wait till the Darknet PR is merged in before merging this.

**Comment by kartikdutt18** [comment]
Agreed, Makes sense.

**Comment by KimSangYeon-DGU** [comment]
@kartikdutt18 After #20 gets merged, please check this PR as well :)

**Comment by kartikdutt18** [comment]
Sure, looks like added some code to the wrong branch. I will remove it. Thanks for pointing it out.

**Comment by KimSangYeon-DGU** [comment]
@kartikdutt18 Thanks for another great contribution :+1:

**Comment by kartikdutt18** [comment]
Great, Thanks @KimSangYeon-DGU, @saksham189, @zoq for the reviews.

**Comment by kartikdutt18** [comment]
Updated the wiki as well.

**Pull request merged**